apps/entry/views.py

```diff
@@ -327,12 +324,17 @@ def get_idu_data(filters=None):
         locations_data = figure_data.pop('locations', None)
         location_parse = extract_location_data(locations_data)
 
+        event_codes_data = figure_data.pop('event_codes', None)
```

Comment:
We should handle None at extract_event_code_data level, but we will need to return empty 'code and 'code_type'
```suggestion
        event_codes_data = figure_data.pop('event_codes', [])
```